src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -58,47 +63,66 @@ public SSHConfig() throws IOException, DuplicateKeyException, YamlReaderExceptio
+                "The default port is 22. Change it if you have a different port setup in your network, are hosting one or more other services on the same port, or are otherwise wanting to use the SSH service on a different port.",
+                "Example:",
+                "port: 22",
+                "Notice: You will likely need to open this port in your firewall settings to allow incoming connections, as well as port-forward it in your router settings if you are hosting the server on a local network.");
 
         auth_method = put(name, "auth-method").setDefValues("key-only")
             .setComments(
                 "Select the authentication method for the SSH console.",
                 "Available options: 'user-pass-only', 'key-only', 'user-pass-key'.",
                 "user-pass-only: Requires a username and password to connect. Requires the username and password fields below.",
                 "key-only: Requires a public key to connect. Requires the allowed-keys-path field below.",
-                "user-pass-key: Requires either a username and password or a public key to connect. Requires all fields below.");
+                "user-pass-key: Requires either a username and password or a public key to connect. Requires all fields below.",
+                "Example:",
+                "auth-method: key-only");
 
-        allowed_keys_path = put(name, "allowed-keys-path")
+        allowed_keys_path = put(name, "allowed-keys-path").setDefValues("./autoplug/allowed_ssh_keys.txt")
             .setComments(
                 "The .txt file containing the public keys that are allowed to connect to the SSH console.",
                 "The file must contain one public key per line.",
                 "The file must be in the OpenSSH format.",
                 "Exaple:",
                 "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDf6... user@host",
                 "Create keys with 'ssh-keygen -t rsa -b 4096' and add them to the file.",
-                "The genereated file can be found in the .ssh directory of the user that created the key, unless a different path was specified.",
-                "The generated file will be a ____.pub file, which contains the public key.");
+                "The generated file can be found in the .ssh directory of the user that created the key, unless a different path was specified.",
+                "The generated file will be a .pub file, which contains the public key.",
+                "Example:",
+                "allowed-keys-path: ./autoplug/allowed_ssh_keys.txt",
+                "Example connection command: `ssh -i /path/to/private/key username@server-ip-address`");
         
-        server_private_key = put(name, "server-private-key")
+        server_private_key = put(name, "server-private-key").setDefValues(System.getProperty("user.dir") + "./ssh/id_rsa")
```

Comment:
This might be platform dependent I guess?
You can set the info based on the platform with OSUtils.IS_WINDOWS for example.
Also are you sure the path is correct? Dir/./ssh